### PR TITLE
OpenSRP 0.2.3 RC5

### PR DESCRIPTION
**Pull request by ndegwamartin**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

   FHIR core release 0.2.3 RC 5

**Engineer Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 

**Code Reviewer Checklist**
- [x] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [x] I have verified any strings visible on UI components are in the `strings.xml` file
- [x] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [x] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Comment by pld** [comment]
k #2327 still a legit test failure

File: docs/developer-setup/automated-releases.mdx [hunk 1]
```diff
@@ -87,21 +87,22 @@ Where the app icon is an image of the appropriate size and the app name is an XM
 
 As part of integrating Continuous Delivery(CD) into the development lifecycle, CI is set up to generate an APK.
 
-The configuration requires the tag to:
-* The product name: _opensrp_
+The configuration requires the tag to have the following parts/sections in order:
+* The prefix `v`
 * Followed by the _version_ in the format `[0-9]+.[0-9]+.[0-9]+[0-9a-zA-Z.-]+`
+* The APK flavor as set up in the gradle build configuration : _quest_
```
Comment (pld) [inline docs/developer-setup/automated-releases.mdx:93]: let's keep quest replaced w/OpenSRP throughout, we want that to be the external name for the default flavor.

Quest is a module name for the UI portion of the code.

We also want other flavors to be prefixed w/opensrp, like opensrp-myproject-versionnumber.apk

**Review by pld — CHANGES_REQUESTED** [review]

**Review by pld — COMMENTED** [review]
not sure why we have those other changes, are those hold-overs from a previous of another branch maybe?

File: docs/developer-setup/automated-releases.mdx (hunk 1 above)
Comment (ndegwamartin) [reply docs/developer-setup/automated-releases.mdx:93]: Will be updating to this

**Comment by ndegwamartin** [comment]
> not sure why we have those other changes, are those hold-overs from a previous of another branch maybe?

Looks like they were introduced by the merge commit from main here : https://github.com/opensrp/fhircore/pull/2376/commits/e87a2bffee7565d6aa161535a49762ff0f77f38f

**Comment by ndegwamartin** [comment]
Closing this in favor of https://github.com/opensrp/fhircore/pull/2409

**Pull request closed without merging**